static/app/components/segmentedControl.tsx

```diff
@@ -12,34 +12,48 @@ import {LayoutGroup, motion} from 'framer-motion';
 import InteractionStateLayer from 'sentry/components/interactionStateLayer';
 import {FormSize} from 'sentry/utils/theme';
 
-export interface SegmentedControlItemProps extends ItemProps<any> {
-  key: React.Key;
+export interface SegmentedControlItemProps<Value extends string> extends ItemProps<any> {
```

Comment:
does it HAVE to be a string? what about numbers?

Reply:
It does, unfortunately, because `AriaRadioGroupProps` expects a string (I figure they want to make it behave the same way native radio inputs do)